XIVComboPlugin/IconReplacer.cs

```diff
@@ -898,6 +916,145 @@ private ulong GetIconDetour(byte self, uint actionID)
+                        return PCT.ClawMuse;
+
+                    if ((PCTGauge.CanvasFlags & CanvasFlags.Maw) != 0)
+                        return PCT.FangMuse;
+                }
+
+                if (actionID == PCT.WeaponMotif)
+                {
+                    if (PCTGauge.WeaponMotifDrawn)
+                        return PCT.StrikingMuse;
+
+                    if (lastMove == PCT.HammerStamp)
+                        return PCT.HammerBrush;
+
+                    if (lastMove == PCT.HammerBrush)
+                        return PCT.HammerPolish;
+
+                    if (SearchBuffArray(PCT.HammerReady))
+                        return PCT.HammerStamp;
+                }
+            }
+
+            if (Configuration.ComboPresets.HasFlag(CustomComboPreset.PictoStarrySkyCombo))
+            {
+                var PCTGauge = JobGauges.Get<PCTGauge>();
+                if (actionID == PCT.LandscapeMotif)
+                {
+                    if (PCTGauge.LandscapeMotifDrawn)
+                        return PCT.StarryMuse;
+                    if (SearchBuffArray(PCT.StarStruck))
+                        return PCT.StarPrism;
+                    return PCT.StarryMotif;
+                }
+            }
+            
+            //VIPER
+            if (Configuration.ComboPresets.HasFlag(CustomComboPreset.ViperDeathRattleMaximum))
+            {
+                if (actionID == VPR.SteelFangs || actionID == VPR.DreadFangs)
+                {
+                    if (iconHook.Original(self, VPR.SerpentsTail) == VPR.DeathRattle)
+                        return VPR.DeathRattle;
+                }
+            }
+
+            if (Configuration.ComboPresets.HasFlag(CustomComboPreset.ViperDeathRattleMaximum))
+            {
+
+                if (actionID == VPR.DreadMaw || actionID == VPR.SteelMaw)
+                {
+                    if (iconHook.Original(self, VPR.SerpentsTail) == VPR.LastLash)
+                        return VPR.LastLash;
+                }
+            }
+            
+            if (Configuration.ComboPresets.HasFlag(CustomComboPreset.ViperDeathRattleMaximum)){
+                if (actionID == VPR.Reawaken && SearchBuffArray(VPR.Buffs.Reawakened))
+                    {
+                        var gauge = JobGauges.Get<VPRGauge>();
+
+                        if (level >= VPR.Levels.Legacies)
+                        {
+                            if (iconHook.Original(self, VPR.SerpentsTail) == VPR.FirstLegacy || 
+                                iconHook.Original(self, VPR.SerpentsTail) == VPR.SecondLegacy ||
+                                iconHook.Original(self, VPR.SerpentsTail) == VPR.ThirdLegacy ||
+                                iconHook.Original(self, VPR.SerpentsTail) == VPR.FourthLegacy)
+                                return iconHook.Original(self, VPR.SerpentsTail);
+                        }
+
+                        var maxtribute = 4;
```

Comment:
`var maxTribute = level >= VPR.Levels.Ouroboros ? 5 : 4;`

Reply:
Changed it ty :)